### PR TITLE
Ensemble trajectory and cv results

### PR DESCRIPTION
**Pull request by jianswang**
@mfeurer Please review this pull request for (loss metrics cv_results_ and indices_files results don't match when ensemble size=1 #626)

### CODE REVIEW

File: autosklearn/automl.py [hunk 1]
```diff
@@ -803,7 +804,10 @@ def sprint_statistics(self):
         sio.write('auto-sklearn results:\n')
         sio.write('  Dataset name: %s\n' % self._dataset_name)
         sio.write('  Metric: %s\n' % self._metric)
-        idx_best_run = np.argmax(cv_results['mean_test_score'])
+        if not self._metric._optimum:
```
[the remark this answers is not in the document]
Comment (jianswang) [reply autosklearn/automl.py]: Yes, you are absolutely right. 
However, I just realised that it's not the most accurate one if there were failed algorithms for loss minimization task. So I have added idx_success above the doubt you were having and at the back of cv_results['mean_test_score']. What do you think?

**Comment by jianswang** [comment]
> Thanks for the contribution @jianswang. The code looks good except for the one comment I left.
> 
> Could you please have a look at the following PEP8 issue: https://travis-ci.org/automl/auto-sklearn/jobs/499136579

It is due to backslash. I tried to remove the backslash in a way that the code is working but it is either giving me the error of code being too long or having whitespace at the back. What do you say?

**Comment by mfeurer** [comment]
Okay, then I think this looks good overall. Thanks a lot for your contribution!

**Comment by mfeurer** [comment]
Oh, I just saw this is against master. I just repointed it against development, let's hope it still works.

**Pull request merged**